src/bin/latency.rs

```diff
@@ -160,27 +172,54 @@ fn do_server(iface_name: String, oneway: bool) {
                 },
             }
         };
-        println!("Received {} bytes", recv_bytes);
-
         // Match packet size
         let mut rx_packet = packet.split_at(recv_bytes as usize).0.to_owned();
-
         let mut eth_pkt = MutableEthernetPacket::new(&mut rx_packet).unwrap();
         if eth_pkt.get_ethertype() != EtherType(ETHERTYPE_PERF) {
             continue;
         }
-
+        let mut perf_pkt = MutablePerfPacket::new(eth_pkt.payload_mut()).unwrap();
         if oneway {
```

Comment:
As we have seperate opcode for oneway mode and rtt mode. We don't need `oneway` parameter. Just see *opcode* and do whatever based on that